src/Main.java

```diff
@@ -1,5 +1,65 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println();
+
+        // Задача 1
+        System.out.println("Задача 1");
+        int year = 2025;
+        leapYear(year);
+
+        // Задача 2
+        System.out.println();
+        System.out.println("Задача 2");
+        int clientOS = 0;
+        int clientDeviceYear = 2021;
+        selectApp(clientOS, clientDeviceYear);
+
+        // Задача 3
+        System.out.println();
+        System.out.println("Задача 3");
+        int deliveryDistance = 95;
+        if (deliveryDistance > 100) {
+            System.out.printf("Доставки нет");
+        } else {
+            System.out.println("Для доставки потребуется дней: "+ deliveryTime(deliveryDistance));
+        }
+    }
+
+    public static void leapYear (int year) {
+        double year4 = year % 4;
+        double year100 = year % 100;
+        double year400 = year % 400;
+        if (year <= 1584) {
+            System.out.println("Григорианский календарь был введён позже");
+        } else if ((year4 == 0 && year100 != 0) || year400 == 0) {
+            System.out.println(year + "  год — високосный год");
+        } else {
+            System.out.println(year + " год — невисокосный год");
+        }
+    }
+
+    public static void selectApp (int clientOS, int clientDeviceYear) {
```

Comment:
названия методов должны начинаться с глагола и кратко описывать, что они (методы) делают. Например,
```suggestion
    public static void printInfoAboutVersion(int clientOS, int clientDeviceYear) {
```